libs/core/src/lib/wizard/wizard.component.ts

```diff
@@ -74,11 +122,53 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
             });
             this._cdRef.detectChanges();
         });
+        this.resizeHandler();
     }
 
     /** @hidden */
     ngOnDestroy(): void {
         this._subscriptions.unsubscribe();
+        this.wrapperContainer.nativeElement.removeEventListener('scroll', handleTimeoutReference);
+    }
+
+    /**
+     * @hidden
+     * @private
+     * This function determines the height of the wizard content by looking for the document's shellbar, the wizard
+     * navigation and the wizard footer, and calculating the height based on their presence.
+     */
+    private _calculateContentHeight(): number {
+        let shellbarHeight = 0,
+            wizardNavHeight = 0,
+            wizardFooterHeight = 0;
+        if (document.querySelector<HTMLElement>('.' + SHELLBAR_CLASS)) {
+            shellbarHeight = document.querySelector<HTMLElement>('.' + SHELLBAR_CLASS).clientHeight;
+        }
+        const wizard = this._elRef.nativeElement.querySelector('.' + WIZARD_CLASS);
+        if (wizard.querySelector('.' + WIZARD_NAVIGATION_CLASS)) {
+            wizardNavHeight = wizard.querySelector('.' + WIZARD_NAVIGATION_CLASS).clientHeight;
+        }
+        if (wizard.querySelector('.' + BAR_FOOTER_CLASS)) {
+            wizardFooterHeight = wizard.querySelector('.' + BAR_FOOTER_CLASS).clientHeight;
+        }
+        return shellbarHeight + wizardNavHeight + wizardFooterHeight;
+    }
+
+    /** @hidden */
+    private _setContainerAndTallContentHeight(): void {
+        const wizard = this._elRef.nativeElement.querySelector('.' + WIZARD_CLASS);
+        const combinedHeight = this.contentHeight ? this.contentHeight : this._calculateContentHeight();
+        if (wizard.querySelector('.' + WIZARD_CONTAINER_WRAPPER_CLASS)) {
+            wizard.querySelector('.' + WIZARD_CONTAINER_WRAPPER_CLASS).style.height =
+                'calc(100vh - ' + combinedHeight + 'px)';
+        }
+        wizard.querySelectorAll('.' + WIZARD_CONTENT_CLASS).forEach((node) => {
+            node.style.height = 'auto';
+        });
+        if (wizard.querySelector('.' + WIZARD_TALL_CONTENT_CLASS)) {
+            wizard.querySelector('.' + WIZARD_TALL_CONTENT_CLASS).style.height =
+                'calc(100vh - ' + combinedHeight + 'px)';
+        }
     }
```

Comment:
Could you create method that will set height property to all of these elements `_setWizardContentHeight`?